oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/migration/version/VersionableEditor.java

```diff
@@ -79,7 +80,7 @@ public class VersionableEditor extends DefaultEditor {
 
     private VersionableEditor(Provider provider, NodeBuilder rootBuilder) {
         this.rootBuilder = rootBuilder;
-        this.versionStorage = getVersionStorage(rootBuilder);
+        this.versionStorage = createVersionStorage(rootBuilder);
```

Comment:
Hm, on secomd thought... I'd like to understand under what circumstances there can be an Oak repository without version storage. Isn't it always created by default?

Reply:
Depends on the RepositoryInitializers used. org.apache.jackrabbit.oak.InitialContent does create it, com.adobe.granite.repository.impl.GraniteContent does not. After all these years using it, I'd guess that's fine.